docs/service-brokers/helm-broker-service-classes/redis/docs/overview.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Overview
+type: Overview
+---
+
+[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server, since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps, and hyperloglogs.
```

Comment:
```suggestion
[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps, and hyperloglogs.
```